### PR TITLE
adding NOJ

### PR DESCRIPTION
**Pull request by ZsgsDesign**
adding NOJ to OJ list

### CODE REVIEW

File: docs/intro/resources.md [hunk 1]
```diff
@@ -42,6 +42,8 @@
     可以方便的在 Vjudge 上提交别的 OJ 的题，尤其是一些国内不太方便的 OJ 。
 -   [ZOJ](http://acm.zju.edu.cn/onlinejudge/)
     浙大
+-   [NOJ](http://acm.njupt.edu.cn/)
+    南京邮电大学在线测评系统，自身拥有题目两千余，通知支持多个国内外OJ的提交，可以直接再NOJ提交别的OJ的题。
```
Comment (Ir1d) [inline docs/intro/resources.md]: typo？

**Comment by TrisolarisHD** [comment]
@ZsgsDesign adding 🤔

File: docs/intro/resources.md (hunk 1 above)
Comment (ZsgsDesign) [reply docs/intro/resources.md]: Nothing wrong???

Comment (TrisolarisHD) [reply docs/intro/resources.md]: fixed

Comment (ZsgsDesign) [reply docs/intro/resources.md]: Oh, thank you.

**Comment by ZsgsDesign** [comment]
@Ir1d I believe the typo has been fixed.

**Comment by Ir1d** [comment]
辛苦啦～ 谢谢

**Pull request merged**